Sources/SQLKit/Query/SQLTableConstraint.swift

```diff
@@ -1,13 +1,17 @@
 public struct SQLTableConstraint: SQLExpression {
-    public var columns: [SQLExpression]
+    /// Serialized after the name and algorithm but before the modifier
+    public var columns: [SQLExpression]?
```

Comment:
I think we want to be using [`SQLColumnConstraint`](https://github.com/vapor/sql-kit/blob/master/Sources/SQLKit/Query/SQLColumnConstraint.swift) here instead.